IDR/production/metadata_extraction/download_jsons/get_json_files.py

```diff
@@ -0,0 +1,68 @@
+import json
+import requests
+import pathlib
+from tqdm import tqdm
+import pandas as pd
+
+if __name__ == "__main__":
```

Comment:
This file inside this directory (very nested) seems like it should contain functions to import, and not one that is so integral as to explicitly run in the terminal.

If it is an important script, then maybe it should be more front and center (so as to not get lost in such long nests). Am I thinking about this script correctly?

Reply:
Ah you're right. I moved these files up a directory and removed the `downloaded_jsons` directory. I meant to have more extraction methods in there, but I don't think that will happen anytime soon.

Added in 402a5f63c5cdad5d8d5b9cb7836d5a92b8c7e7cf.